### PR TITLE
Simplify vbproj files

### PR DESCRIPTION
**Pull request by dmonroym**
There's a lot of code in these files that doesn't seem to be necessary. Deleting it as it will improve readability and makes moving code around in #6652 a lot easier

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/project-system/pull/6766)

### CODE REVIEW

**Comment by drewnoakes** [comment]
Have you tested whether any of these elements are added back to the project when editing these items in their designers?

**Comment by dmonroym** [comment]
I had left the Designers open and modifying them there didn't re-add them to the vbproj but if I close everything the incorrect icons and Designers won't show up anymore. Thanks for catching that @drewnoakes - I'm abandoning this

File: src/Microsoft.VisualStudio.Editors/Microsoft.VisualStudio.Editors.vbproj [hunk 1]
```diff
@@ -24,570 +24,9 @@
...
-    <None Include="Docs\custom_vs_system3.png" />
-    <None Include="Docs\CustomColorRenames.txt" />
-    <None Include="Docs\CustomColors.txt" />
-    <None Include="Docs\Project Designer Architecture Visual.png" />
-    <None Include="Docs\Project Designer Architecture.png" />
-    <None Include="Docs\Project Designer Architecture.vsd" />
-    <None Include="Docs\ProjectDesigner Control Flow.doc" />
-    <None Include="MyExtensibility\MyExtensibilityClassDiagram.cd" />
-    <None Include="PropPages\AdvBuildSettingsPropPage.ComboItem.vb" />
   </ItemGroup>
```
Comment (drewnoakes) [inline src/Microsoft.VisualStudio.Editors/Microsoft.VisualStudio.Editors.vbproj:30]: These can probably still be removed.

File: src/Microsoft.VisualStudio.Editors/Microsoft.VisualStudio.Editors.vbproj [hunk 2]
```diff
@@ -24,570 +24,9 @@
...
-      <SubType>Form</SubType>
-    </Compile>
-    <Compile Update="XmlToSchema\WebUrlDialog.Designer.vb">
-      <DependentUpon>WebUrlDialog.vb</DependentUpon>
-      <SubType>Form</SubType>
-    </Compile>
-    <Compile Update="XmlToSchema\WebUrlDialog.vb">
-      <SubType>Form</SubType>
-    </Compile>
-    <Compile Update="XmlToSchema\Wizard.vb" />
```
Comment (drewnoakes) [inline src/Microsoft.VisualStudio.Editors/Microsoft.VisualStudio.Editors.vbproj:576]: This line looks redundant.

**Pull request closed without merging**